src/popup/components/InputAmountV2.vue

```diff
@@ -61,76 +61,81 @@
   </InputField>
 </template>
 
-<script>
-import { mapGetters } from 'vuex';
-import { pick } from 'lodash-es';
+<script lang="ts">
+import {
+  computed,
+  defineComponent,
+  onMounted,
+  PropType,
+  watch,
+} from '@vue/composition-api';
+import BigNumber from 'bignumber.js';
+import { useGetter } from '../../composables';
+import { IAsset } from '../../types';
+import { rxJsObservableToVueState } from '../utils';
 import InputField from './InputField.vue';
 import InputSelectAsset from './InputSelectAsset.vue';
 
-export default {
+export default defineComponent({
   components: {
     InputSelectAsset,
     InputField,
   },
   props: {
     value: { type: [String, Number], default: '' },
     label: { type: String, default: null },
-    selectedAsset: { type: Object, default: null },
+    selectedAsset: { type: Object as PropType<IAsset | null>, default: null },
     aeOnly: Boolean,
     showTokensWithBalance: Boolean,
   },
-  subscriptions() {
-    return pick(this.$store.state.observables, [
-      'balance',
-      'balanceCurrency',
-    ]);
-  },
-  data() {
-    return {
-      aeToken: null,
-    };
-  },
-  computed: {
-    ...mapGetters('fungibleTokens', ['getAeternityToken']),
-    ...mapGetters([
-      'formatCurrency',
-      'account',
-    ]),
-    hasError() {
-      return this.$validator.errors.has('amount');
-    },
-    totalAmount() {
-      return (this.currentAsset?.current_price)
-        ? ((+this.value || 0) * this.currentAsset.current_price).toFixed(2)
-        : 0;
-    },
-    currentTokenFiatPrice() {
-      return this.currentAsset?.current_price?.toFixed(2);
-    },
-    currentAsset() {
-      return this.selectedAsset || this.aeToken;
-    },
-  },
-  watch: {
-    hasError(value) {
-      return this.$emit('error', value);
-    },
-  },
-  created() {
-    this.aeToken = this.getAeternityToken({
-      tokenBalance: this.balance,
-      balanceCurrency: this.balanceCurrency,
+  setup(props, { root, emit }) {
+    // eslint-disable-next-line no-unused-vars
+    const getAeternityToken = useGetter<(o: any) => IAsset>('fungibleTokens/getAeternityToken');
+    const formatCurrency = useGetter('formatCurrency');
+
+    const balance = rxJsObservableToVueState<BigNumber>(
+      (root.$store.state as any).observables.balance,
+    );
+    const balanceCurrency = rxJsObservableToVueState<number>(
+      (root.$store.state as any).observables.balance,
+    );
```

Comment:
`balance` and `balanceCurrency` are exactly the same. I think it should look like this

```suggestion
    const balance = rxJsObservableToVueState<BigNumber>(
      (root.$store.state as any).observables.balance,
    );
    const balanceCurrency = rxJsObservableToVueState<number>(
      (root.$store.state as any).observables.balanceCurrency,
    );
```

Reply:
Uuuu, my bad. Thanks for finding this out.